src/usr/local/www/system_camanager.php

```diff
@@ -250,6 +253,12 @@
 
 		$ca['descr'] = $pconfig['descr'];
 
+		if (!isset($pconfig['trusted']) || empty($pconfig['trusted'])) {
+			$ca['trusted'] = false;
+		} else {
+			$ca['trusted'] = $pconfig['trusted'];
+		}
+
```

Comment:
Option set/not set rather that true/false is the way it is done throughout pfSense.  Yes using isset instead of true/false works, is cleaner/clearer and is the consistent boolean option technique in pfSense config.